SokolTextGame/Locations.cs

```diff
@@ -1,15 +1,32 @@
-﻿namespace SokolTextGame
+﻿using System.Runtime.CompilerServices;
```

Comment:
Please keep an eye on these automatic imports from Visual Studio. They only pollute your codebase and don't do anything at all. Please remove this one.